### ISSUE TITLE
Wrong transfer with Tensorflow

### ISSUE THREAD

**Issue by sonfiree** [bonlime/keras-deeplab-v3-plus#71]
Your model returns as follows.
512x512x21- for each pixel of the prediction probability
Tensorflow
512Х512 - Each pixel image class

How to make a conclusion 512x512?

**Comment by bonlime** [comment]
Take argmax on the last axis
